packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -461,8 +468,7 @@ export class ProjectActions {
       throw Error(`Cannot create spec without activeProject.`)
     }
 
-    const config = await this.ctx.project.getConfig(project.projectRoot)
-    const integrationFolder = config.integrationFolder || project.projectRoot
+    const integrationFolder = 'cypress/integration' || project.projectRoot
```

Comment:
This will need to be updated in another PR, it's around the logic to scaffold specs from the unified runner UI - trying to keep this PR smallish.